package.json

```diff
@@ -26,32 +26,31 @@
     "test": "npm run lint && npm run spec"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.0.0"
   },
   "dependencies": {
     "debug": "^4.1.0",
-    "electron-installer-common": "^0.6.1",
-    "fs-extra": "^7.0.1",
-    "glob": "^7.1.3",
+    "electron-installer-common": "^0.6.3",
+    "fs-extra": "^8.0.1",
+    "glob": "^7.1.4",
     "glob-promise": "^3.4.0",
```

Comment:
I've found that `glob-promise` isn't really needed, you just need to `promisify(glob)`.